tests/unittests/core/mongodb_test.py

```diff
@@ -107,12 +107,19 @@ def test_overwrite_partial_uri(self, monkeypatch):
 
     def test_singleton(self):
         """Test that MongoDB class is a singleton."""
-        orion_db = MongoDB(username='user', password='pass', name='orion_test')
+        orion_db = MongoDB('mongodb://localhost',
+                           port=27017, name='orion', username='lala',
+                           password='pass')
         # reinit connection does not change anything
         orion_db.initiate_connection()
         orion_db.close_connection()
         assert MongoDB() is orion_db
 
+    def test_change_server_timeout(self):
+        """Test that the server timeout is correctly changed."""
+        orion_db = MongoDB(username='user', password='pass', name='orion_test', server_timeout=6000)
+        assert 6000 in str(orion_db)
```

Comment:
What we would like to test here, is that server_timeout actually affects the timeout. You can test with 1000 and verify that `time.time() - start` is smaller than 2000 and try again with 5000 to make sure it gets adapted.

Reply:
Woopsie

Reply:
```suggestion
        assert '6000' in str(orion_db)
```